packages/hl7/src/connection.ts

```diff
@@ -17,14 +17,16 @@ export type Hl7MessageQueueItem = {
 export class Hl7Connection extends Hl7Base {
   readonly socket: net.Socket;
   readonly encoding: string;
+  readonly enhancedMode: boolean;
```

Comment:
Is this property used anywhere?

Reply:
Not currently, but the intent would be for introspection (ie. for debugging, being able to see what a given connection's `enhancedMode` is set to)